cli/cmd/suppressions.go

```diff
@@ -63,4 +68,183 @@ func init() {
 	// gcp
 	suppressionsCommand.AddCommand(suppressionsGcpCmd)
 	suppressionsGcpCmd.AddCommand(suppressionsListGcpCmd)
+	suppressionsGcpCmd.AddCommand(suppressionsMigrateGcpCmd)
+}
+
+func autoConvertSuppressions(convertedPolicyExceptions []map[string]api.PolicyException) {
+	cli.StartProgress("Creating policy exceptions ...")
```

Comment:
Extreme nitpick, but I don't think we usually leave space between the ellipsis
```suggestion
	cli.StartProgress("Creating policy exceptions...")
```